opentreemap/treemap/tests/test_views.py

```diff
@@ -1955,3 +1955,45 @@ def test_instance_list_only_public(self):
         other_instance.save()
 
         self.assertEqual(2, len(public_instances_geojson(make_request())))
+
+
+class UserAutocompleteTest(OTMTestCase):
+    def setUp(self):
+        self.i1 = make_instance()
+        self.i2 = make_instance()
+
+        self.mike = make_user(instance=self.i1, username='mike')
+        self.maria = make_user(instance=self.i1, username='Maria')
+        self.matt = make_user(instance=self.i2, username='MATT')
+
+    def assert_users_in_list(self, users_list, *users):
+        self.assertEqual(len(users_list), len(users))
+
+        for i in range(0, len(users)):
+            user_dict = users_list[i]
+            user = users[i]
+            self.assertIn('id', user_dict)
+            self.assertIn('username', user_dict)
+            self.assertEqual(user.pk, user_dict['id'])
+            self.assertEqual(user.username, user_dict['username'])
+
+    def test_full_results(self):
+        users_list = users(make_request(), self.i2)
+        self.assert_users_in_list(users_list, self.matt)
+
+    def test_filtering(self):
+        users_list = users(make_request(), self.i1)
```

Comment:
Seems to me that using this request in `test_full_results` would spare having to repeat it here and in `test_max`.